hw/top_earlgrey/lint/top_earlgrey_fpv_lint_cfgs.hjson

```diff
@@ -0,0 +1,103 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+
+  // This is the primary cfg hjson for FPV linting. It imports ALL individual lint
+  // cfgs of the IPs FPV environments.
+  // This enables to run them all as a regression in one shot.
+  name: top_earlgrey_fpv_batch
+
+  import_cfgs:      [// common server configuration for results upload
+                     "{proj_root}/hw/data/common_project_cfg.hjson"
+                     // tool-specific configuration
+                     "{proj_root}/hw/lint/tools/dvsim/{tool}.hjson"]
+
```

Comment:
Can you also add this rel path here and add a link to the [HW dashboard](hw/_index.md)?
```
  // Different dashboard output path for each tool
  rel_path: "hw/top_earlgrey/fpv/lint/{tool}"
```

Reply:
Oh Thanks Michael, that is what this path is for.. Thanks and will do now!

Reply:
Do I need to add to some nightly lint regression script?

Reply:
Hey Cindy, yes I'll send you the path to the file directly. It is not in this repository, though.